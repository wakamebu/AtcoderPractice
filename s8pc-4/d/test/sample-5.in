12
1 2
2 3
2 4
4 5
5 6
5 7
6 8
8 9
2 10
10 11
11 12
