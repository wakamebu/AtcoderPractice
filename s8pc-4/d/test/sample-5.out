3.666666666667
2.250000000000
3.666666666667
2.833333333333
2.555555555556
2.666666666667
4.333333333333
2.666666666667
5.333333333333
2.500000000000
2.500000000000
5.000000000000
